css/style.css

```diff
@@ -0,0 +1,68 @@
+   body{
+        background-color: lightblue;
+        text-align: center;
+        
+      }
+
+      h1 {
+        color: blue;
+        align-items: center;
```

Comment:
- Add properties of width and height.

Reply:
- [ ] It is always nice to make your comments suggestive or having polite phrases, instead of commanding sentences. Also, make sure you explained well why this requested change is important.